Source/UI/BillingForm/Default Implementation/BillingForm/Header/Main/DefaultBillingFormHeaderCellStyle.swift

```diff
@@ -1,9 +1,8 @@
 import UIKit
 
 public struct DefaultBillingFormHeaderCellStyle: BillingFormHeaderCellStyle {
-
     public var backgroundColor: UIColor = .clear
-    public var headerLabel: ElementStyle = DefaultHeaderLabelFormStyle()
+    public var headerLabel: ElementStyle = DefaultHeaderLabelFormStyle(text: Constants.LocalizationKeys.BillingForm.Header.title)
```

Comment:
None of the other properties inject anything to the default values, why did you make this one inject the text?

Reply:
Inject the `text` to `DefaultHeaderLabelFormStyle`

Reply:
Yes, but what I'm confused is why we don't inject anything to `Default...`, but this is the first one we inject.

Also a default component just injects something to another default component that previously wasn't needed. I'm just trying to see whether it was a mistake commit or if there was some genuine need that meant we have to do so now.

Reply:
`DefaultHeaderLabelFormStyle` is common Style Implementation for header views, it has shared fonts and colors. Anything can be injected like text, font, color, etc. This change is for the text for billing form header view.